### PR TITLE
Support Md, multi-paragraph dashboard descriptions

### PR DESCRIPTION
**Pull request by openjck**

### CODE REVIEW

**Comment by openjck** [comment]
To test it, replace *src/config.json* with the following and run *npm start*.

**src/config.json**
```
{
    "dashboards": [
        {
            "key": "user-activity",
            "menuTitle": "User Activity",
            "source": "https://openjck-ensemble-transposer-dev.s3.amazonaws.com/datasets/desktop/user-activity",
            "supportsRegions": true
        },
        {
            "key": "usage-behavior",
            "menuTitle": "Usage Behavior",
            "source": "https://openjck-ensemble-transposer-dev.s3.amazonaws.com/datasets/desktop/usage-behavior",
            "supportsRegions": true
        },
        {
            "key": "hardware",
            "menuTitle": "Hardware",
            "source": "https://openjck-ensemble-transposer-dev.s3.amazonaws.com/datasets/desktop/hardware",
            "supportsRegions": false
        }
    ],
    "nextButtons": [
        {
            "from": "/",
            "to": "/dashboard/user-activity",
            "text": "Proceed to User Activity"
        },
        {
            "from": "/dashboard/user-activity",
            "to": "/dashboard/usage-behavior",
            "text": "Proceed to Usage Behavior"
        },
        {
            "from": "/dashboard/usage-behavior",
            "to": "/dashboard/hardware",
            "text": "Proceed to Hardware"
        }
    ]
}
```

File: src/components/views/Dashboard.jsx [hunk 1]
```diff
@@ -46,21 +47,40 @@ export default props => {
...
-            <p id="dashboard-description">{props.description}</p>
-        );
+        const multipleParagraphs = Array.isArray(props.description);
+        const markdownParser = markdownIt('zero').enable(['emphasis']);
+
+        if (multipleParagraphs) {
+            maybeDescription = (
+                <div id="dashboard-description">
+                    {props.description.map(paragraph => (
+                        <p key={paragraph} dangerouslySetInnerHTML={
```
Comment (spasovski) [inline src/components/views/Dashboard.jsx]: seems like a really long key, if the order of these in the DOM won't change it should be alright to key the index

**Comment by spasovski** [comment]
npm react-markdown seems decent also but it's my first time using it - it's possible this pkg is better.

r+ either way the key comment isn't a big deal i think

**Comment by openjck** [comment]
> npm react-markdown seems decent also but it's my first time using it - it's possible this pkg is better.

Nice! I'll consider that moving forward. I've been using markdown-it in this repo for a while but maybe it's not the best choice.

**Comment by openjck** [comment]
Hey, this is a neat service. No clear winner, but interesting nonetheless.

https://www.npmtrends.com/markdown-it-vs-react-markdown

**Comment by spasovski** [comment]
hah neat, guess you picked the more popular one...either way i don't have much experience with markdown libs so will see

File: src/components/views/Dashboard.jsx (hunk 1 above)
Comment (openjck) [reply src/components/views/Dashboard.jsx]: Good call. I had read that indices should not be used as keys, but upon further reading this seems to be a perfect exception to the rule. 👍

**Pull request merged**